build/netradiant/README.md

```diff
@@ -0,0 +1,21 @@
+Unvanquished support for Netradiant
```

Comment:
Could also have moved this file instead of creating a new one with the same content. 😉 

Also, better phrasing wouldn't hurt here, though that goes for a lot of the english text in this repository…